packages/ui/src/services/AuthService.ts

```diff
@@ -25,7 +25,7 @@ export default interface AuthService {
 
   isAuthenticated(): Promise<Boolean>;
 
-  getUser(): Promise<User>;
+  getUser(): Promise<User | null>;
```

Comment:
I had the same issue, but @AbnerZheng managed to fix it. Do you remember that?

Reply:
@zie1ony / @AbnerZheng - does this need changed urgently or could we merge and then fix later (if needs be)?  

![Screenshot from 2021-01-08 16-03-48](https://user-images.githubusercontent.com/69711689/104036362-38515f80-51cb-11eb-94cb-20065bd3ed4c.png)

Reply:
No, we can do it later. I have found you merge the master and resolve conflict several times. Let's merge it firstly.

Reply:
No, we can do it later. I have found you merge the master and resolve conflict several times. Let's merge it firstly.

Reply:
Sounds good